x-pack/plugins/apm/server/lib/service_map/get_service_map.ts

```diff
@@ -129,38 +124,32 @@ async function getServicesData(options: IEnvOptions) {
         [SERVICE_NAME]: bucket.key as string,
         [AGENT_NAME]:
           (bucket.agent_name.buckets[0]?.key as string | undefined) || '',
-        [SERVICE_ENVIRONMENT]: options.environment || null,
-        [SERVICE_FRAMEWORK_NAME]:
-          (bucket.service_framework_name.buckets[0]?.key as
-            | string
-            | undefined) || null
+        [SERVICE_ENVIRONMENT]: options.environment || null
       };
     }) || []
   );
 }
 
 function getAnomaliesData(options: IEnvOptions) {
-  const { client } = options.setup;
+  const { start, end, client } = options.setup;
+  const rangeQuery = { range: rangeFilter(start, end, 'timestamp') };
```

Comment:
I don't think extracting `rangeQuery` from the query improves the readability. I also don't think we should reuse the `rangeFilter` in this case. It's more readable to write it out in full:

```ts
{ 
  range: { timestamp: { gte: start, lte: end, format: 'epoch_millis' } } 
}
```